stdlib/tkinter/dialog.pyi

```diff
@@ -12,5 +12,5 @@ DIALOG_ICON: str
 class Dialog(Widget):
     widgetName: str
     num: int
-    def __init__(self, master: Incomplete | None = None, cnf: Mapping[str, Any] = {}, **kw: Incomplete) -> None: ...
+    def __init__(self, master: Incomplete | None = None, cnf: Mapping[str, Any] = {}, **kw) -> None: ...
```

Comment:
For files like this, where there is only a single "bare Incomplete" in the file, I'd prefer it if we `noqa`'d it if it means we can keep running the stricter pyright settings on the file as a whole

Alternatively, I suppose we could also consider using `# pyright: ignore[reportUnknownParameterType,reportMissingParameterType]` on this line, rather than adding the whole file to the excludelist in the stricter pyright config file.

Reply:
I actually prefer that we list those files in the stricter config. After all, complaining about missing annotations is the only point of the stricter config, so it makes sense not to lie to it about it.

Reply:
But by excluding whole files from the stricter config, we increase the risk that new functions and methods that are lacking annotations accidentally "slip in" without us noticing it. If significant portions of the file are unannotated, then I agree that we should just exclude the file, but I think opting into the stricter config where possible is helpful to maintain the quality of our stubs

Reply:
But we still have manual reviews for that. I don't like "lying" about things like that and making it harder to spot areas for improvement.

Reply:
I think it's pretty easy for a reviewer to miss an unannotated argument slipping in -- if it wasn't, we wouldn't have any need for the stricter pyright config at all :)

And, I don't see this as "lying". We'd still need to explicitly add a `noqa` or `pyright: ignore` comment on the offending line, which stands out on the page and can be easily grepped for.

Reply:
Well, I disagree. Adding extra ignores is worse to me than adding those files to the stricter excludes.

Reply:
(That said, I've never been a fan of the stricter pyright config, and I think it's a mistake we added it.)